src/irc.nim

```diff
@@ -564,6 +586,8 @@ proc newAsyncIrc*(address: string, port: Port = 6667.Port,
               realname = "NimBot", serverPass = "",
               joinChans: seq[string] = @[],
               msgLimit: bool = true,
+              ssl: bool = false,
+              sslContext = getDefaultSSL(),
               callback: proc (irc: AsyncIrc, ev: IrcEvent): Future[void]
```

Comment:
Huh, I'm actually surprised that this is allowed (non-default argument after default arguments).